source/treeInterceptorHandler.py

```diff
@@ -20,10 +20,17 @@
 import vision
 from speech.types import SpeechSequence
 from controlTypes import OutputReason
+import extensionPoints
 
 if TYPE_CHECKING:
 	import NVDAObjects
 
+# Extension points action:
+# Triggered when browseMode state is changed.
+# Callback definition: Callable(browseMode: bool) -> None
+# browseMode - True when current treeInterceptor.passThrough is False.
+# False otherwise, or if focus is not in a treeInterceptor.
+post_browseModeStateChange = extensionPoints.Action()

```

Comment:
can this get more detailed documentation on usage in-line with other extension points?

For example see `pre_writeCells`